PersonApi/serverless.yml

```diff
@@ -143,11 +143,12 @@ resources:
         RetrievalRoleArn: ${ssm:/housing-tl/${self:provider.stage}/app-config/retrieval-role-arn}
         LocationUri: ssm-parameter:///housing-tl/person-api/configurationData
     DeploymentStrategy:
-      Type: AWS::AppConfig::AllAtOnce
+      Type: AWS::AppConfig::DeploymentStrategy
       Properties:
         Name: "DeploymentStrategy"
-        DeploymentDurationInMinutes: 3
+        DeploymentDurationInMinutes: 0
         GrowthFactor: 10
```

Comment:
Should the growth factor be `100` ?

Reply:
Yes it should be! Thank you for spotting it out